src/etl/ETLSource.cpp

```diff
@@ -1026,17 +983,17 @@ ETLSourceImpl<Derived>::forwardToRippled(
         //
         // https://github.com/ripple/rippled/blob/develop/cfg/rippled-example.cfg
         ws->set_option(websocket::stream_base::decorator(
-            [&request](websocket::request_type& req) {
+            [&request,&clientIp] (websocket::request_type& req) {
                 req.set(
                     http::field::user_agent,
                     std::string(BOOST_BEAST_VERSION_STRING) +
                         " websocket-client-coro");
                 req.set(
                     http::field::forwarded,
```

Comment:
I'm assuming this is using the `Forwarded` request header w/ `"for=1.2.3.4".

Does rippled support this, or does it only support `X-Forwarded-For`?

Reply:
This is how we do it in reporting mode: https://github.com/ripple/rippled/blob/89766c5f218e49f038e30aee05c1be9216f78bf5/src/ripple/app/reporting/ETLSource.cpp#L858